rfcs/repo-management/shield-issue-expectations.md

```diff
@@ -0,0 +1,99 @@
+
+#### Proposed solution
+
+We should actively encourage the issues' authors to create PRs that solve their issues. This should be done by both PM and Engineering so we can adopt a more successful open-source model and shift some of the burden off the person that is on Shield at the time.
+
+#### Problem
+
+Engineering and PM sometimes do the same task twice, which leads to duplicate efforts and wasted time.
+
+#### Proposed solution
+
+This is an issue that is not as apparent as others but some tasks are done twice sometimes by PM and Engineering. These include but are not limited to checking if a bug is reproducing or if certain behavior is a bug or a regression. This should be done only once, especially in issues where it is clear that the described behavior is buggy, and communicated well between the two disciplines.
+
+#### Problem
+
+There are issues logged in that lack the complete information needed to address them.
+
+#### Proposed solution
+
+We have templates in place for bugs, accessibility requests, feature requests, etc. There might be an area of opportunity to improve them but we could also be more strict about only addressing those issues that have all the information asked for in the templates.
+
+#### Problem
+
+The expectation on which issues should be fixed immediately vs can be delayed vs will probably not get fixed in the short or medium term is not well defined.
+
+#### Proposed solution
+
+We should agree and write down which issues should definitely be fixed while a person is on Shield and which ones are not necessarily going to be fixed during this time frame. Here is a proposed set of expectations:
+
+> _Issues that need to be fixed ASAP:_
+>
+> - Accessibility issues that affect our accessibility score card.
+> - Bugs identified as high impact or blocking.
+> - Bugs identified as regressions of previous functionality
+>
+> _Issues for which we encourage contributions that we are happy to help guide and review:_
+>
+> - Bugs that are identified as not being high impact.
+> - Feature requests that have been discussed about and agreed on a solution with team members.
+>
+> _Issues that we need to address but can't currently:_
+>
+> - Bugs that require a breaking change in terms of the API interface and require a major version release.
+>
+> _Issues that we make no commitments on ever fixing:_
+>
+> - Feature requests that have not been discussed with team members.
+
+#### Problem
+
+Users of FluentUI that are logging issues in our repository have a different set of expectations regarding issue resolution than the ones we have as part of our Shield process.
+
+#### Proposed solution
+
+We should clearly communicate with our customers how we are looking at issues and set clear expectations on them. This would ideally be in the way of a wiki entry in our repository detailing our Shield process with the expectations on issue resolution there.
```

Comment:
👍 that would be great not just for consumers but for anyone within core team to adhere to same standards